setup.py

```diff
@@ -1,36 +1,35 @@
+try:
+    from setuptools import setup, find_packages
+except ImportError:
+    from distutils.core import setup
+
 import subprocess
 import sys
-from setuptools import setup, find_packages
 import versioneer
 
-
-# Utility method(s)
-def install(package):
-    subprocess.check_call(
-        [sys.executable, "-m", "pip", "install", "--user", package]
-    )
-
-
 # Requirements definitions
 SETUP_REQUIRES = [
-    "setuptools == 59.5.0",
+    "setuptools==62.3",
 ]
 
 INSTALL_REQUIRES = [
-    "sqlalchemy",
-    "pandas>=1.1.0",
-    "numpy",
-    "timer",
-    "tqdm",
-    "torch-cluster==1.5.9",
+    "sqlalchemy>=1.4.37",
+    "pandas>=1.4",
+    "numpy>=1.22",
+    "timer>=0.2",
+    "tqdm>=4.64",
+    "dill>=0.3",
+    "wandb>=0.12",
+    "matplotlib>=3.5",
+    "scikit_learn~=1.1.1",
```

Comment:
Is there a reason, like for `torch` below, for using `~=` rather than  `>=`? For `torch` I figure it has to do with the CUDA version, but would it be a problem if someone manually (for whatever reason) wanted to go with `torch=1.10+cu113` rather than `torch=1.11+cu115`, as long as the pytorch code itself is compatible?